cli_tools_e2e_test/go.mod

```diff
@@ -5,13 +5,11 @@ go 1.13
 require (
 	cloud.google.com/go/storage v1.10.0
 	github.com/GoogleCloudPlatform/compute-image-tools/cli_tools v0.0.0-20200813223603-3672ca27e050
+	github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common v0.0.0
 	github.com/GoogleCloudPlatform/compute-image-tools/daisy v0.0.0-20200813213118-5b8ac20eab97
 	github.com/GoogleCloudPlatform/compute-image-tools/go/e2e_test_utils v0.0.0-20200813223603-3672ca27e050
 	github.com/aws/aws-sdk-go v1.34.4
-	golang.org/x/net v0.0.0-20200813134508-3edf25e44fcc // indirect
-	golang.org/x/tools v0.0.0-20200813203630-136574234359 // indirect
 	google.golang.org/api v0.30.0
-	google.golang.org/genproto v0.0.0-20200813001606-1ccf2a5ae4fd // indirect
-	google.golang.org/grpc v1.31.0 // indirect
-	honnef.co/go/tools v0.0.1-2020.1.5 // indirect
 )
+
+replace github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common => ./common
```

Comment:
This won't work on server side, it only works from local. Also no need to require its sub directory per my understanding.

Reply:
By 'server side', were you referring to building on Cloud Build? If so, I've been building the Dockerfile `gce_image_import_export_tests.Dockerfile` on Cloud Build to run tests locally.

Here `common` refers to a go module. It happens to be a subdirectory, and that's made explicit using `replace`. To double check this for your comment, I removed the require and replace for `github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common`. This resulted in a build error of: 

```
Step #0: test_suites/import/cli_tests.go:28:2: module github.com/GoogleCloudPlatform/compute-image-tools@latest found (v0.0.0-20200810192041-68836ecfc7e3), but does not contain package github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common/assert
Step #0: test_suites/import/cli_tests.go:29:2: module github.com/GoogleCloudPlatform/compute-image-tools@latest found (v0.0.0-20200810192041-68836ecfc7e3), but does not contain package github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common/compute
Step #0: test_suites/import/import_tests.go:38:2: module github.com/GoogleCloudPlatform/compute-image-tools@latest found (v0.0.0-20200810192041-68836ecfc7e3), but does not contain package github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common/logging
Step #0: test_suites/export/image_export_test_suite.go:27:2: module github.com/GoogleCloudPlatform/compute-image-tools@latest found (v0.0.0-20200810192041-68836ecfc7e3), but does not contain package github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common/storage
Step #0: test_suites/export/image_export_test_suite.go:28:2: module github.com/GoogleCloudPlatform/compute-image-tools@latest found (v0.0.0-20200810192041-68836ecfc7e3), but does not contain package github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common/utils
Step #0: error building image: error building stage: failed to execute command: waiting for process to exit: exit status 1
```

Reply:
Hmm I didn't notice 'common' has been a module